test-data/src/test/resources/org/openmrs/module/fhir2/api/dao/impl/FhirMedicationRequestDaoImpl_initial_data.xml

```diff
@@ -80,15 +86,22 @@
     <drug drug_id="2003" concept_id="4022" dosage_form="51" name="Rifampicin" combination="true" creator="1"
           date_created="2005-02-24 00:00:00.0" retired="false" uuid="yuf00b94-26fe-102b-80bc-0017a4787b12"/>
 
-    <drug_order order_id="1005" drug_inventory_id="2001" duration="6" duration_units="27" dispense_as_written="true"
```

Comment:
Editing of the existing dataset normally is not a good idea, I would rather add new ones. But since all tests passed, I think you are good.

Reply:
@corneliouzbett i will take care next time and thanks for review.